spotify_player/src/config/theme.rs

```diff
@@ -284,6 +289,14 @@ impl Theme {
             Some(s) => s.style(&self.palette),
         }
     }
+    pub fn secondary_row(&self) -> tui::style::Style {
+        match &self.component_style.secondary_row {
+            None => Style::default()
+                .bg(StyleColor::DarkGray)
```

Comment:
can use `StyleColor::Rgb` instead if you want a specific color that is not available in the palette? My suggestion would be using default theme if `None`, so it doesn't cause any surprise for current users.

Reply:
Correct me if I'm wrong, but this enables the alternating row colors by default, right? I was thinking that should be the intended behavior; should it instead only set alternating row colors when using a custom theme?

Reply:
> I was thinking that should be the intended behavior; should it instead only set alternating row colors when using a custom theme?

Yeah, it should be opt-in. My reasoning is that setting to some default style (e.g dark gray as above) might "break" some users' theme. If one wants the alternating colors, they can update the style in their configs.